src/tests/ftest/util/soak_utils.py

```diff
@@ -66,11 +66,13 @@ def add_pools(self, pool_names, ranks=None):
     target_list = ranks if ranks else None
     for pool_name in pool_names:
         path = "".join(["/run/", pool_name, "/*"])
-        properties = self.params.get('properties', path, "")
+        properties = self.params.get('properties', path, None)
         # allow yaml pool property to override the scrub default; whether scrubber is enabled or not
-        if self.enable_scrubber and "scrub" not in properties:
+        if self.enable_scrubber and "scrub" not in str(properties):
```

Comment:
This works because `str(None) -> 'None' (literally a string containing "None"). But it's better practice to do something like
```suggestion
        if self.enable_scrubber and properties is not None and "scrub" not in properties:
```